src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -21,7 +22,19 @@ interface Props {
   rootIri?: string;
 }
 
+enum RadioButtonType {
+  Predicted = "Predicted",
+  Manual = "Manual",
+  Operational = "Operational",
+}
+
+enum ManualFailureRateType {
+  Sns = "Sns",
+  External = "External",
+}
+
 const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }: Props) => {
+  if (shapeToolData) console.log("shapeToolData", shapeToolData);
```

Comment:
this console.log does not belong to the PR

Reply:
@Kasmadei any solution for that?

Reply:
Was added by mistake (for debug). Removed.